lib/p2p/packets/types/SwapFailedPacket.ts

```diff
@@ -7,18 +7,18 @@ export type SwapErrorPacketBody = {
   errorMessage: string;
 };
 
-class SwapErrorPacket extends Packet<SwapErrorPacketBody> {
+class SwapFailedPacket extends Packet<SwapErrorPacketBody> {
```

Comment:
Per my last comment, let's leave the packet renaming changes out for now because they are a breaking change for p2p communication.